code/datums/shuttles.dm

```diff
@@ -147,50 +130,34 @@
 	suffix = "supermatter"
 	name = "Hyperfractal Gigashuttle"
 	description = "\"I dunno, this seems kinda needlessly complicated.\"\n\
-		\"This shuttle has very a very high safety record, according to \
-		Centcom Officer Cadet Yins.\"\n\
-		\"Are you sure?\"\n\
-		\"Yes, it has a safety record of N-A-N, which is apparently \
-		larger than 100%.\""
-	admin_notes = "Supermatter that spawns on shuttle is special anchored \
-		'hugbox' supermatter that cannot take damage and does not take in \
-		or emit gas. Outside of admin intervention, it cannot explode. \
-		It does, however, still dust anything on contact, emits high levels \
-		of radiation, and induce hallucinations in anyone looking at it \
-		without protective goggles. Emitters spawn powered on, expect \
-		admin notices, they are harmless."
+	\"This shuttle has very a very high safety record, according to Centcom Officer Cadet Yins.\"\n\
+	\"Are you sure?\"\n\
+	\"Yes, it has a safety record of N-A-N, which is apparently larger than 100%.\""
+	admin_notes = "Supermatter that spawns on shuttle is special anchored 'hugbox' supermatter that cannot take damage and does not take in or emit gas. \
+	Outside of admin intervention, it cannot explode. \
+	It does, however, still dust anything on contact, emits high levels of radiation, and induce hallucinations in anyone looking at it without protective goggles. \
+	Emitters spawn powered on, expect admin notices, they are harmless."
 
 /datum/map_template/shuttle/emergency/imfedupwiththisworld
 	suffix = "imfedupwiththisworld"
 	name = "Oh, Hi Daniel"
-	description = "How was space work today? \
-		Oh, pretty good. We got a new space station and the company will make a lot of money. \
-		What space station? \
-		I cannot tell you; it's space confidential. \
-		Aw, come space on. Why not? \
-		No, I can't. Anyway, how is your space roleplay life?"
+	description = "How was space work today? Oh, pretty good. We got a new space station and the company will make a lot of money. What space station? I cannot tell you; it's space confidential. \
```

Comment:
that's because that one used both actual quotes and newlines this is literally all on one line